flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/source/MySqlSourceTestBase.java

```diff
@@ -86,4 +89,11 @@ public static void assertEqualsInOrder(List<String> expected, List<String> actua
         assertEquals(expected.size(), actual.size());
         assertArrayEquals(expected.toArray(new String[0]), actual.toArray(new String[0]));
     }
+
+    public static void assertExecutorIsTerminated(DebeziumReader<?, ?> reader) throws Exception {
+        Field executorField = reader.getClass().getDeclaredField("executor");
```

Comment:
Use reflection is not cool here, if you really want to check `executorService` member of `SnapshotReader`, you can expose a method in `SnapshotReader` like:

```java
    @VisibleForTesting
    public ExecutorService getExecutorService() {
        return executorService;
    }
```